applications/crossbar/priv/couchdb/schemas/system_config.kazoo_endpoint.json

```diff
@@ -25,6 +25,11 @@
             "description": "kazoo_endpoint default ignore completed elsewhere",
             "type": "boolean"
         },
+        "default_rtcp_mux": {
```

Comment:
FreeSwitch default value is 'false' this setting reflect it is.
This this value nothing is changed in current kazoo clusters.